vision.md

```diff
@@ -0,0 +1,407 @@
+# What is our vision?
+
+The aim of creating thin-edge.io is to provide a IoT edge device framework for
+IoT project teams which makes it easy to enable resource constrained devices for
+IoT. Unlike other solutions, we are not just another single-purpose agent but a
+flexible platform with re-usable components without any vendor lock-in, focused
+on adressing the needs of both IT and OT users.
+
+## Our motivation
+
+We believe that IoT (edge/device management, middleware, data analytics) are the
+driving forces of the fourth industrial revolution: companies are forced to
+leverage IoT solutions to stay competitive. In the past a large amount of
+companies have suffered from failed IoT projects and initiatives, where a lot of
+time and money was spent on device enablement: connectivity, security, device
+management, etc...
+
+Thats why we are creating thin-edge.io. It provides an easy and flexible way for
+IoT project teams and device manufactures to make their devices IoT-ready
+without having to re-invent the wheel with costly and time-consuming,
+potentially embedded, development projects and without having to reevaluate
+existing projects or applications to enable them with thin-edge.io.
+
+## Our target segments
```

Comment:
It seems that when it comes to device integration / thin-edge.io all three target segments listed here have very similar needs. I believe it would be rather interesting to look at the target industries, use-cases (remote monitoring, software management, ML model execution, ?) and target device types, thin-edge.io is designed for (IoT Gateways, PLCs, etc.) and which it is not designed for (e.g. micro-controllers, LPWAN sensors, battery constrained devices?, Industrial PCs?, etc.)

Reply:
In this section the idea is to focus on the target segments and their most important needs. I will check if we can incorporate deeper statements regardign monitoring, software management and ML topics and device types in later sections.